packages/components/Callout/macos/Callout.swift

```diff
@@ -321,4 +378,30 @@ private class FlippedVisualEffectView: NSVisualEffectView {
 	}
 }
 
+
+/// NSEvent localMonitors are an _old_ API that requires you to do some  memory management yourself.
+/// You _must_ call `removeMonitor`exactly once per monitor created from. This helper class enforces
+/// that with the use of an extra boolean to track whether we have already allocated/released our monitor.
+/// https://developer.apple.com/documentation/appkit/nsevent/1533709-removemonitor
+private class GuardedEventMonitor: Any {
```

Comment:
nit should we prefix with FRN or something just in case?

Reply:
another nit: can we create a new file for this private class?

Reply:
This is Swift, so we don't have that namespace clashing consideration to deal with.

Reply:
Would we want another file for the FlippedVisualEffectView above as well?